Dockerfile

```diff
@@ -1,13 +1,23 @@
 FROM quay.io/centos/centos:stream8
 RUN dnf -y install dnf-plugins-core &&\
     dnf -y config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo &&\
-    dnf -y install golang docker-ce-cli
+    dnf -y install golang docker-ce-cli git make
```

Comment:
It looks like this should be a multi-stage Dockerfile so that `git` and `make` don't stick around in the container.

Reply:
I believe in a multistage containerfile, both the builder stage (with the test stage) and the final stage depend on `flake8`, so that code would be duplicated. I think the generation of the test files should go into CI somewhere, but I was unable to setup a vm with golang 1.18. If the test stage is removed and put into CI, then I think the multistage containerfile would make more sense.